stellarphot/settings/models.py

```diff
@@ -435,18 +449,50 @@ class Observatory(BaseModelWithTableRep):
 
     """
 
-    name: str
+    name: Annotated[str, Field(description="Name of the observatory")]
     latitude: Annotated[
-        Latitude, _UnitQuantTypePydanticAnnotation, BeforeValidator(add_degree_to_float)
+        Latitude,
+        _UnitQuantTypePydanticAnnotation,
+        BeforeValidator(add_degree_to_float),
+        Field(
+            description="Latitude of the observatory",
+            examples=[
+                "46.8738",
+                "46.8738 degree",
+                "46:52:25.68 degree",
+                "46d52m25.68s",
+            ],
+        ),
     ]
     longitude: Annotated[
         Longitude,
         _UnitQuantTypePydanticAnnotation,
         BeforeValidator(add_degree_to_float),
+        Field(
+            description="Longitude of the observatory",
+            examples=[
+                "-96.7678",
```

Comment:
I know it is a little silly, but given the target audience includes new users, should we clarify negative is used for W longitude and the 360 version is always E of Greenwich?  It would make the description too long, so maybe we put this in the documentation or in the Python notebooks themselves?

Reply:
I'm thinking an example. in the documentation -- for now I'll add a bit of discussion to the `Examples` section of the `Observatory` docstring.